index.src.html

```diff
@@ -760,10 +760,10 @@ <h4 id="interfaces-credential-types-passwordcredential"><code>PasswordCredential
   :   <dfn attribute for="PasswordCredential">additionalData</dfn>
   ::  If the developer wishes to specify additional data to insert into the
       request body when submitting the credential information to a remote
-      endpoint, they can do so by assigning a {{FormData}} or
-      {{URLSearchParams}} object to this attribute. The credential information
-      will be mixed into the object to produce the body. The value is `null`
-      unless otherwise specified.
+      endpoint, they can do so by assigning a {{FormData}},
+      {{URLSearchParams}} or a generic object to this attribute. The
```

Comment:
> Is serializability a language-independent counterpart of what JSON stringification is in JavaScript?

Yes. But we're looking at making this more clearly JSON-specific in WebIDL.

> If so, the Credential Manager API spec needs, in addition to requesting serializability, define how the user agent inserts the password value (and username).

Right.